### PR TITLE
feature : Redirect to our github organization [fixes #230]

### LINKED ISSUE #230: [FEATURE] Redirect to our github organization
### Detailed description

We should add a destination to this button :
image

### Context

This button is displayed but do nothing.

### Possible implementation

Use this snippet in the file `src/components/Global/Navbar.js` Line 56:

```html
// Update the href above!

<a
  href="#" 
  className="inline-flex items-center px-4 py-2 text-base font-medium text-gray-600 bg-gray-100 border border-transparent rounded-md hover:text-white/80 hover:bg-gray-700"
>
  GitHub
  <SiGithubsponsors className="w-5 h-5 ml-3 text-pink-600" aria-hidden="true" />
</a>
```

* Link of the organization is: https://github.com/WebXDAO

Use this link for the `href` param.

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by najeebkp**
## Related Issue

- Redirect to our github organization #230

Closes: #230

### Describe the changes you've made

Just added the href url

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested manually

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/231"></a>

### CODE REVIEW

**Comment by najeebkp** [comment]
Sure @vinzvinci . but please make sure you merge it before hacktoberfest ends :)

**Comment by mkubdev** [comment]
We should be good :)

**Comment by vinzvinci** [comment]
> Sure @vinzvinci . but please make sure you merge it before hacktoberfest ends :)

Yaah, don't worry hehehe.

**Comment by mkubdev** [comment]
@vinzvinci Do we have update about this?

**Comment by vinzvinci** [comment]
> @vinzvinci Do we have update about this?

Hey kub, can u use your current sponsor direct link here? we need to setup a bank org. for this if ever.

File: src/components/Global/Navbar.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const Navbar = () => {
             </div>
 
             <a
-              href="#"
+              href="https://github.com/WebXDAO"
```
Comment (mkubdev) [inline src/components/Global/Navbar.js:57]:
```suggestion
              href="https://github.com/sponsor/mkubdev"
```

**Review by mkubdev — CHANGES_REQUESTED** [review]
Added suggestion for the sponsor page in waiting the WebXDAO one.

**Comment by vinzvinci** [comment]
Hey, thanks for your pull request. I apologize for the delay in approving the sponsor page (org. sponsor is not approve last year). I hope you have a successful Hacktoberfest! Closing this as we are preparing for rebranding. Thank you for your contributions, and we hope to see more from you in the future!

**Pull request closed without merging**